src/trex/quality_control.py

```diff
@@ -10,25 +10,7 @@
 from tinyalign import hamming_distance
 
 
-def load_reads(data_dir: pathlib.Path) -> pd.DataFrame:
-    """Loads saved reads into a DataFrame"""
-    read_dir = data_dir / "reads.txt"
-    return pd.read_csv(read_dir, delimiter="\t")
-
-
-def load_molecules(data_dir: pathlib.Path) -> pd.DataFrame:
-    """Loads saved molecules before correcting into a DataFrame."""
-    mols_dir = data_dir / "molecules.txt"
-    return pd.read_csv(mols_dir, delimiter="\t")
-
-
-def load_molecules_corrected(data_dir: pathlib.Path) -> pd.DataFrame:
```

Comment:
Isn't it good to keep this in case reading these files changes in the future?
I found it comfortable not having to wonder how to read these files and only calling the read functions.

Reply:
It would be nice to have one place where all these file names are defined. But at the moment, the name `"molecules.txt"` is still hardcoded in the `write_reads_or_molecules` functions in both `run10x.py` and `smartseq3.py`. If the name changes there, you will still have to adjust this function and haven’t actually gained anything.

The way I would suggest to do it is to introduce a constant (like `MOLECULES_PATH = "molecules.txt"`) that gets used everywhere. But that would be a different PR, and I’m still skeptical.

I see what you mean and generally agree with you, but in this case the readability of `pd.read_table(... / "molecules.txt")` was more important for me.

Reply:
So are you against merging this or would it be ok if I leave adding a `readers` module up to a subsequent PR from you?

Reply:
In favour of merging and a subsequent PR for ```readers```.